amgut/webserver.py

```diff
@@ -112,9 +111,9 @@ def __init__(self):
 def main():
     startup_tests()
     # replace spaces for underscores to autocomplete easily in a shell
-    # format looks like american_gut_8888_2014-10-06_15:30:20.256035.log
-    prefix = ("%s_%d_%s.log" % (media_locale['LOCALE'], options.port,
-                                str(datetime.now()))).replace(' ', '_')
+    # format looks like american_gut_8888.log
+    prefix = ("/var/log/%s_%d.log" % (media_locale['LOCALE'], options.port,
+                                      )).replace(' ', '_')
```

Comment:
:+1:

On Mon, Mar 28, 2016 at 10:42 AM, Joshua Shorenstein <
notifications@github.com> wrote:

> In amgut/webserver.py
> <https://github.com/biocore/american-gut-web/pull/580#discussion_r57602843>
> :
>
> > @@ -112,9 +111,9 @@ def __init__(self):
> >  def main():
> >      startup_tests()
> >      # replace spaces for underscores to autocomplete easily in a shell
> > -    # format looks like american_gut_8888_2014-10-06_15:30:20.256035.log
> > -    prefix = ("%s_%d_%s.log" % (media_locale['LOCALE'], options.port,
> > -                                str(datetime.now()))).replace(' ', '_')
> > +    # format looks like american_gut_8888.log
> > +    prefix = ("/var/log/%s_%d.log" % (media_locale['LOCALE'], options.port,
> > +                                      )).replace(' ', '_')
>
> append
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/american-gut-web/pull/580/files/7900541bf91370ec056d8bcefae35f4646c5b9c6#r57602843>
>